src/lib/pages/backoffice/Item/ItemForms/ItemEditor/index.js

```diff
@@ -1 +1,17 @@
-export { ItemEditor } from './ItemEditor';
+import { connect } from 'react-redux';
+
+import { fetchPastLoans } from '../../ItemDetails/ItemPastLoans/state/actions';
+import ItemEditorComponent from './ItemEditor';
+
+const mapStateToProps = state => ({
+  itemPastLoans: state.itemPastLoans.data.total,
+});
+
+const mapDispatchToProps = dispatch => ({
+  fetchPastLoans: itemPid => dispatch(fetchPastLoans(itemPid)),
```

Comment:
I am a bit worried by adding this, yet another search, but I guess we don't have much options...
Anyway, it will happen quite rarely...
Make sure that this search finishes before rendering the form, it is blocking, maybe?

Reply:
This search will happen any time we open the form edit.
Now it's updated to wait for the search to finish before rendering the form, thanks!